napari/_qt/widgets/_tests/test_qt_layerlist.py

```diff
@@ -160,6 +160,33 @@ def test_removing_layers(qtbot):
     assert check_layout_dividers(view.vbox_layout, len(layers))
 
 
+def test_clearing_layerlist(qtbot):
+    """Test clearing layer list."""
+    layers = LayerList()
+    view = QtLayerList(layers)
+
+    qtbot.addWidget(view)
+
+    layer_a = Image(np.random.random((10, 10)))
+    layer_b = Image(np.random.random((15, 15)))
+    layer_c = Image(np.random.random((15, 15)))
+    layer_d = Image(np.random.random((15, 15)))
+    layers.append(layer_a)
+    layers.append(layer_b)
+    layers.append(layer_c)
+    layers.append(layer_d)
```

Comment:
```suggestion
    layers.extend([Image(np.random.random((15, 15))) for _ in range(4)])
```
... or is it important that the first layer be `(10, 10)`?